src/FixedDataTable.js

```diff
@@ -775,12 +768,13 @@ class FixedDataTable extends React.Component {
           fixedRightColumns={fixedRightColumnGroups}
           scrollableColumns={scrollableColumnGroups}
           visible={true}
-          onColumnResize={this._onColumnResize}
+          onColumnResizeEnd={onColumnResizeEndCallback}
           onColumnReorder={onColumnReorder}
           onColumnReorderMove={this._onColumnReorderMove}
           showScrollbarY={scrollEnabledY}
           scrollbarYWidth={scrollbarYWidth}
           isRTL={this.props.isRTL}
+          isHeader={true}
```

Comment:
Needed to pass the extra props only to header cells and also for backwards compatibility to spawn ResizerKnob on the header cells.